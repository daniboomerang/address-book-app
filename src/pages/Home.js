import React from 'react';

/**
 * Adreess book home page
 * It retrieves the users from from the https://randomuser.me API and renders the users grid
 * It provides a search in order to filter results by first and last name
 */
const Home = () => (
  <div className="w-full h-full justify-center flex items-center">Address book home page</div>
);

export default Home;
