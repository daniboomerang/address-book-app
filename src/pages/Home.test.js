import React from 'react';
import { mount } from 'enzyme';
import Home from './Home';

describe('Home page component', () => {
  it('should render correctly', () => {
    const wrapper = mount(<Home />);

    expect(wrapper.text()).toBe('Address book home page');
    expect(wrapper).toMatchSnapshot();
  });
});
