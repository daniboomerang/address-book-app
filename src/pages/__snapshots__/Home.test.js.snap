// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page component should render correctly 1`] = `
<Home>
  <div
    className="w-full h-full justify-center flex items-center"
  >
    Address book home page
  </div>
</Home>
`;
